src/main/java/com/rocketden/main/model/Language.java

```diff
@@ -0,0 +1,40 @@
+package com.rocketden.main.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.rocketden.main.exception.LanguageError;
+import com.rocketden.main.exception.api.ApiException;
+
+import org.apache.commons.lang.WordUtils;
+
+import lombok.Getter;
+
+@Getter
+public enum Language {
+
+    PYTHON,
+    RUBY,
+    SWIFT,
```

Comment:
There's already a CodeLanguage enum class - could you add this driverGeneratorName logic to that class and use that one instead? Or delete the old one and refactor the codebase to use this class, whichever is easier

Reply:
Whoops, that was silly of me, I forgot about CodeLanguage; thanks for catching this, I'll use that instead.

Reply:
Done! I accidentally changed on #103 at first instead, but the change is on both now.